### DISCUSSION TITLE
How to Identify is node is intersecting/overlapping edge? or Want to spilt node if I drop node on it.

### DISCUSSION THREAD

**Discussion by savalemilind6984** [xyflow/xyflow#4032] [Q&A]
Hi Team,

In my application, I want to split an edge if I drop a node on it. 
So how to identify the node overlapping/intersecting edge like "isNodeInterSecting" (isNodeIntersecting works for node not for an edge)
eg. suppose there are 2 nodes "A"and ""B" and they both are connected with and edge. If I drop node "C" on edge then want connection edge between "A"---"C"---"B".

If anyone have implement or any solution please suggest.

Thanks, in advance.

**Comment by stevegg** [comment]
I'm looking for this same capability.
